### PR TITLE
Update and rename RP-0.netkan to RP-1.netkan

### PR DESCRIPTION
**Pull request by Bornholio**
remove CTT, add RP-0 to conflict, change name to Realistic Progression One

### CODE REVIEW

File: RP-1.netkan [hunk 1]
```diff
@@ -46,5 +45,6 @@
     ],
     "conflicts" : [
         { "name" : "CrowdSourcedScience" }
+        { "name" : "Realistic Progression Zero" }
```
Comment (Olympic1) [inline RP-1.netkan]: Should be the identifier, so `RP-0`

File: RP-1.netkan [hunk 2]
```diff
@@ -17,7 +17,6 @@
         { "name" : "CustomBarnKit" },
         { "name" : "DeadlyReentry" },
         { "name" : "VenStockRevamp" },
-        { "name" : "CommunityTechTree" },
         { "name" : "KerbalConstructionTime" },
         { "name" : "RealSolarSystem" }
     ],
```
Comment (Olympic1) [inline RP-1.netkan]: Add following mods to the depends according to the readme in #864 
```
{ "name" : "ProceduralParts" },
{ "name" : "DMagicOrbitalScience" }
```

Remove in recommends
```
{ "name" : "ProceduralParts" },
```

Rename in recommends
`RCSBuildAid` to `RCSBuildAidCont`
`HangarExtender` to `HangerExtenderExtended`
`TextureReplacer` to `TextureReplacerReplaced`

**Comment by Bornholio** [comment]
Thanks @Olympic1

File: RP-1.netkan [hunk 3]
```diff
@@ -29,22 +30,23 @@
...
         { "name" : "TestFlight" },
         { "name" : "ConnectedLivingSpace" },
-        { "name" : "HangarExtender" },
+        { "name" : "HangerExtenderExtended" },
         { "name" : "KSP-AVC" },
         { "name" : "MechJeb2" },
         { "name" : "B9-PWings-Fork" },
         { "name" : "ProceduralFairings" },
         { "name" : "ProceduralFairings-ForEverything" },
         { "name" : "ProceduralParts" },
```
Comment (Olympic1) [inline RP-1.netkan]: Remove `ProceduralParts` from recommends as its a dependency

**Comment by Olympic1** [comment]
Just one more thing to fix and this is good for merge

**Comment by Theysen** [comment]
So the name is now set in stone? Just curious, I've been out of the loop for a time now and last is me remembering NK saying something about working name, not yet decided etc :)

**Comment by Bornholio** [comment]
No this is only if RP-0 and development features need to release separately.  else if change it to RP-0 and merge.

**Comment by Bornholio** [comment]
since RP-1 will roll in closing this

**Pull request closed without merging**